OverlayPlugin.Core/MemoryProcessors/Combatant/Common.cs

```diff
@@ -100,6 +100,9 @@ public class Combatant
         public ObjectType Type;
         public MonsterType MonsterType;
         public ObjectStatus Status;
+        public bool IsVisible;
+        [NonSerialized]
+        public byte[] DrawObject;
```

Comment:
Why bother declaring this as a property on the Combatant object at all if it's not going to be serialized (and therefore won't be passed down to overlay handler calls)?

It would be better to either:
a. Make the `DrawObject` struct serializable as well, and then initialize that on the `Combatant` object
b. Completely exclude this property from the `Combatant` object

Reply:
removed from `Combatant`